Confuser.Renamer/Analyzers/VTableAnalyzer.cs

```diff
@@ -46,7 +46,18 @@ public class VTableAnalyzer : IRenamer {
 					// case here.
 					if (!TypeEqualityComparer.Instance.Equals(slot.MethodDef.DeclaringType, type)) {
 						SetupOverwriteReferences(service, modules, slot, type);
-						//CreateOverrideReference(service, slot.MethodDef, slot.Overrides.MethodDef);
+
+						// If required, create the sibling references, so the names of the interfaces line up correctly.
+						var existingReferences = service.GetReferences(slot.MethodDef);
+						var overrideDef = existingReferences
+							.OfType<MemberOverrideReference>()
+							.Where(r => !MethodEqualityComparer.CompareDeclaringTypes.Equals(r.BaseMemberDef as MethodDef, slot.Overrides.MethodDef))
+							.FirstOrDefault();

```

Comment:
More optimal (at least merge `Where` and `FirstOrDefault`):

```suggestion
						var overrideDef = (MemberOverrideReference)existingReferences
							.FirstOrDefault(r => r is MemberOverrideReference memberOverrideReference &&
							 !MethodEqualityComparer.CompareDeclaringTypes.Equals(memberOverrideReference.BaseMemberDef as MethodDef, slot.Overrides.MethodDef));
```

Reply:
Merging `Where` and `FirstOrDefault` → Alright.
But what is the issue  with using `OfType<>` ? It's much cleaner to read.

Reply:
To get rid of using `yield`, it's better for performance. But in general never mind.